src/cryptonotecore/Core.cpp

```diff
@@ -2154,14 +2344,18 @@ namespace CryptoNote
 
 	uint64_t futureTimeLimit;
 
-	if (previousBlockIndex + 1 >= CryptoNote::parameters::DIFFICULTY_TARGET_V2_HEIGHT)
-	{
-		futureTimeLimit = CryptoNote::parameters::DIFFICULTY_TARGET * 6;
-	}
-	else
-	{
-		futureTimeLimit = CryptoNote::parameters::DIFFICULTY_TARGET_V2 * 6;
-	}
+        if (previousBlockIndex + 1 >= CryptoNote::parameters::DIFFICULTY_TARGET_V3_HEIGHT)
+        {
+            futureTimeLimit = CryptoNote::parameters::DIFFICULTY_TARGET_V3 * 6;
+						        }
+        else if (previousBlockIndex + 1 >= CryptoNote::parameters::DIFFICULTY_TARGET_V2_HEIGHT)        
```

Comment:
You'll need to verify the chain syncs with this change. If you want to sort out the misordering of the v1/v2, but still be safe, could change the DIFFICULTY_TARGET_V2 to the V1 value and visa versa?

Reply:
yeah, had to reverse it as it was, otherwise we are not able to sync...